src/common/options.js

```diff
@@ -371,19 +372,8 @@ const options = {
     setRoutingStrategies: (locationParam, newUrl, productType, isUnbxdKey, replace) => {
         if (locationParam === newUrl) {
             return;
-        } else if (productType === "CATEGORY") {
-            /** Do not navigate to base category page  */
-            if (!isUnbxdKey) {
-                history.replaceState(null, "", newUrl);
-            } else {
-                history.pushState(null, "", newUrl);
-            }
         } else {
-            if ((history.state && history.state.replace) || replace) {
-                history.replaceState(null, "", newUrl);
-            } else {
-                history.pushState(null, "", newUrl);
-            }
+            history.pushState(null, "", newUrl);
```

Comment:
you can change this function to
if (locationParam !== newUrl) {
        history.pushState(null, "", newUrl);
}